react-ui/src/components/AppointmentList/AppointmentList.js

```diff
@@ -44,7 +44,13 @@ export default class AppointmentList extends Component {
       });
   }
 
-  async cancelAppointment(event) {
```

Comment:
Please note that `async` is not needed in this function as `.then` was used instead for code consistency.